Assets/HoloToolkit/Input/Scripts/BasicCursor.cs

```diff
@@ -1,94 +0,0 @@
-﻿// Copyright (c) Microsoft Corporation. All rights reserved.
-// Licensed under the MIT License. See LICENSE in the project root for license information.
```

Comment:
Because it isn't needed anymore. @paseb made a new ICursor interface + cursor scripts that are better designed, and include the functionalities tha the old cursor provided